services/viva-ms/README.md

```diff
@@ -0,0 +1,35 @@
+<h1> HELSINGBORG IO SLS VIVA MICRO SERVICE</h1>
+
+## Purpose
+
+The purpose of Viva Micro Serivce is to provide the data flow between the aws and viva api adapter.
+
+## Description
+
+When a case changes status from ongoing to submitted, the microservice responds and sends a request to the adapter, which in turn creates a case in Viva.
+
```

Comment:
Maybe a more general description of the purpose of this integration, since there will be more additions to this integration. 

Proposal: 
The purpose of this microservice is to integrate functionality towards a machine that communicates with the internal bussiness-system VIVA. We call this machine the VIVA Adapter (VADA).

Reply:
Will be updated when the time come